src/Build/Instance/TaskRegistry.cs

```diff
@@ -229,17 +285,20 @@ internal Toolset Toolset
             where I : class, IItem
         {
             ErrorUtilities.VerifyThrowInternalNull(directoryOfImportingFile, nameof(directoryOfImportingFile));
+#if DEBUG
+            ErrorUtilities.VerifyThrow(!taskRegistry._isInitialized, "TaskFactoryDoubleInitialize");
```

Comment:
Do we need a localized string for this, if it's a debug-only internal error? Can this be `VerifyThrowInternal` with an unlocalized string instead?